<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Email_func_birth</class>
 <widget class="QWidget" name="Email_func_birth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>年月日查询</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
	border-image: url(:/background/屏幕截图 2024-06-01 180551.png);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
    <property name="text">
     <string>生日查询</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>1</y>
      <width>32</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="year_sb">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>1</y>
      <width>47</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="day_sb">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>1</y>
      <width>47</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>1</y>
      <width>32</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>月份</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>32</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>年份</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="month_sb">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>1</y>
      <width>47</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);
</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
